src/flows/pipes/Notification/view.tsx

```diff
@@ -704,11 +478,7 @@ ${ENDPOINT_DEFINITIONS[data.endpoint]?.generateTestQuery(data.endpointData)}`
         </FlexBox>
         <Panel.Footer justifyContent={JustifyContent.FlexEnd}>
           <FlexBox margin={ComponentSize.Medium}>
-            <ExportTaskButton
-              generate={generateTask}
-              text="Export Alert Task"
-              type="alert"
```

Comment:
since we're removing some of the props from this, I would check out other usages of this button (not the new one) to see if we can clean this up a bit more. If it's only being used in one other place, we could reduce the props we're sending down and isolate things a little more